README.md

```diff
@@ -13,6 +13,16 @@ designing government services.**
 
 Make sure you have [Node.js](https://nodejs.org/en/) installed.
 
+Note: You will need the [active LTS (Long-term support)](https://github.com/nodejs/Release#release-schedule) Node.js version for this project (as specified in [.nvmrc](./.nvmrc))
+
+### Using nvm
+If you work across multiple Node.js projects there's a good chance they require different Node.js and npm versions.
+
+To make this we use [nvm (Node Version Manager)](https://github.com/creationix/nvm) to switch between versions easily.
+
+1. [install nvm](https://github.com/creationix/nvm#installation)
+2. Run `nvm install` in the project directory (this will use [.nvmrc](./.nvmrc))
```

Comment:
Does this shim `npm` on your `$PATH` etc or do you then need to run a different command when e.g. running start etc?

Reply:
pretty sure they're separate concepts - `nvm install` will look at `.nvmrc` and install and use the correct version of Node/npm. npm would still be used for all the usual commands

Reply:
npm comes along with node, when you install nvm it'll take priority over a current installation